src/mon/OSDMonitor.cc

```diff
@@ -2019,8 +2021,8 @@ bool OSDMonitor::do_prune(MonitorDBStore::TransactionRef tx)
   ceph_assert(num_pruned > 0);
 
   bufferlist bl;
-  osdmap_manifest.encode(bl);
-  tx->put(get_service_name(), "osdmap_manifest", bl);
+  manifest.encode(bl);
+  tx->put(get_service_name(), "manifest", bl);
```

Comment:
probably didn't mean to change the key name?

Reply:
oops. that's what you get for a blanket replace. thanks for catching that. :)